supply/lib/supply/uploader.rb

```diff
@@ -152,11 +152,17 @@ def upload_mapping(apk_version_codes)
     end
 
     def upload_bundles
-      aab_path = Supply.config[:aab]
-      return [] unless aab_path
+      aab_paths = [Supply.config[:aab]] unless (aab_paths = Supply.config[:aab_paths])
+      # return [] unless aab_paths
```

Comment:
@leandroBorgesFerreira Can we remove this commented out line? 🙃

Reply:
fixed =]